pkg/agent/util/net.go

```diff
@@ -165,9 +165,11 @@ func GetIPNetDeviceByName(ifaceName string) (v4IPNet *net.IPNet, v6IPNet *net.IP
 		if ipNet, ok := addr.(*net.IPNet); ok {
 			if ipNet.IP.IsGlobalUnicast() {
 				if ipNet.IP.To4() != nil {
+					if v4IPNet == nil {
```

Comment:
Thanks for catching the bug.

Here your change lets `v4IPNet` and `v6IPNet` take the first IPv4/IPv6 address in `addrList`. Maybe choose the last one also works and makes code simpler? What's your idea? @tnqn

Reply:
The first IP is normally the primary IP and `ifconfig` only lists the first one. So IMO choosing the first one makes more sense.

Reply:
Thanks. 

/test-windows-e2e /test-windows-networkpolicy